### ISSUE TITLE
Support for AGP 8.2.0 

### ISSUE THREAD

**Issue by har5hit** [adobe/aepsdk_flutter#85]
### Expected Behaviour
Library should be easily integrated in project using Android Gradle plugin 8.2.0.

### Actual Behaviour
Library throwing error due to non compatibility with AGP 8.2.0 

### Reproduce Scenario 

Create a sample Flutter project with `flutter_aepcore` library and update AGP in android build.gradle with `classpath 'com.android.tools.build:gradle:8.2.0'` and gradle version in `gradle-wrapper.properties` file to 8.2, after that when trying to build apk, it will throw below error
```
A problem occurred configuring project ':flutter_aepcore'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.
     
     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.
```

#### Solution
Please set namespace in build.gradle following steps in https://d.android.com/r/tools/upgrade-assistant/set-namespace

**Comment by cacheung** [comment]
@egonm12  We don't have a timeline for supporting this yet.  We are aware of this requirement.  
Since this is an Open Source project, you are welcome to make the updates.

**Comment by egonm12** [comment]
@cacheung I opened a pull request #88

**Comment by cacheung** [comment]
@egonm12 Thanks for the contribution, we are discussing the PR internal and will get back to you once a decision is made.

**Comment by egonm12** [comment]
@cacheung any updates?

**Comment by cacheung** [comment]
@har5hit @egonm12 , the namespace support for Android has been released with aepsdk_flutter plugins[ version 4.0.1](https://github.com/adobe/aepsdk_flutter/releases).

@egonm12 , thanks again for your contribution!

We are going to close this ticket.  Please feel free to open a new one if you have more questions.
